src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -221,11 +221,21 @@ export class NavigationTracker extends BaseTracker {
    * Determines if the given element is a header link
    *
    * @param {string} element - The HTML link element to get the type of.
-   * @return {boolean} Returns true if the header tag contains this element, false otherwise.
+   * @return {boolean} Returns true if the header or nav tag contains this element, false otherwise.
    */
   isHeaderMenuBarLink(element: HTMLElement): boolean {
-    const header = document.getElementsByTagName("header")[0];
-    return header && header.contains(element);
+    const header = document.querySelector("header");
+    const nav = document.querySelector("nav");
+
+    if (header && header.contains(element)) {
+      return true;
+    }
+
+    if (nav && nav.contains(element)) {
+      return true;
+    }
+
+    return false;
```

Comment:
We can refactor this down a little. Using option chaining (`?`), we can remove the assertion at the beginning to shorten the if statements to `header?.contains(element))`.

We can go further and refactor the return itself to
`return (header?.contains(element) || (nav?.contains(element));`